R/predictions.R

```diff
@@ -11,7 +11,7 @@
 #' TODO: Write details about how this is done (reference to paper)
 #'
 #' @examples
-#' TODO: Add simple examples
+#' # TODO: Add simple examples
```

Comment:
Extra #?

Reply:
The code below `@examples` should be actual code, and `TODO: Add simple examples` will error, but `# TODO: Add simple examples` will work fine (since it's just commented code).